src/store/configureStore.js

```diff
@@ -1,29 +1,39 @@
+import AsyncStorage from '@react-native-community/async-storage';
 import { createStore, applyMiddleware, compose } from 'redux';
 import thunkMiddleware from 'redux-thunk';
 import { createLogger } from 'redux-logger';
-import _ from 'lodash';
+import { persistStore, persistReducer } from 'redux-persist';
 import AppReducer from 'reducers';
 
 /* eslint-disable */
 const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
 /* eslint-enable */
 
+const persistConfig = {
+  key: 'root',
+  storage: AsyncStorage,
+  whitelist: ['session'],
+};
+
 export default function configureStore(initialState) {
   const middlewares = [thunkMiddleware];
 
   if (__DEV__) {
     const logger = createLogger({
       collapsed: true,
-      predicate: (getState, { type }) => !_.startsWith(type, '@@redux-form'),
     });
     middlewares.push(logger);
   }
 
+  const persistedReducer = persistReducer(persistConfig, AppReducer);
+
   const store = createStore(
-    AppReducer,
+    persistedReducer,
```

Comment:
Do we want to persist every piece of the app state by default?

Reply:
Nope, we have a whitelist in `cofingureStore.js` that specifies which reducers are going to be persisted, by default I just left `session`